apps/betterangels-backend/shelters/enums.py

```diff
@@ -43,19 +45,21 @@ class ImmediateNeedChoices(models.TextChoices):
 
 class GeneralServiceChoices(models.TextChoices):
     CASE_MANAGEMENT = "Case Management", _("Case Management")
+    CHILDCARE = "Childcare", _("Childcare")
     COMPUTERS = "Computers", _("Computers")
+    FINANCIAL_LITERACY_ASSISTANCE = "Financial Literacy/Assistance", _("Financial Literacy/Assistance")
+    HOUSING_NAVIGATION = "Housing Navigation", _("Housing Navigation")
```

Comment:
global comment: do we want these enum values to be human readable? that is, `"Housing Navigation`" vs `"housing_navigation"`?